website/docs/function_utils.md

```diff
@@ -34,9 +34,9 @@ formatted_parameters = build_value_by_func_parameters('key_{param_1}_{param_2}_{
 print(formatted_parameters) # key_1_2_hello world
 
 formatted_parameters = build_value_by_func_parameters('key_{param_1}_{param_2}_{param_3}', function_to_format, 1) 

```

Comment:
who is `function_to_format` ?what parameteres dose it recieved. do it have default values?

Reply:
the function whose params will be formatted, it is on line 30, it has both types of parameters